docs/content/migration.md

```diff
@@ -0,0 +1,545 @@
+| nav-item           |   [Docs][87] |   [Docs][88] | `mat-tab-link`                                         |
+| optgroup           |   [Docs][89] |   [Docs][90] |                                                        |
+| option             |   [Docs][91] |   [Docs][23] |                                                        |
+| progress-linear    |   [Docs][55] |   [Docs][12] | `mat-progress-bar`                                     |
+| progress-circular  |   [Docs][56] |   [Docs][11] | `mat-progress-spinner`                                 |
+| radio-button       |   [Docs][57] |   [Docs][4]  |                                                        |
+| radio-group        |   [Docs][92] |   [Docs][93] |                                                        |
+| select             |   [Docs][59] |   [Docs][23] |                                                        |
+| select-on-focus    |   [Docs][94] |   -          | Not implemented                                        |
+| sidenav            |   [Docs][60] |   [Docs][6]  |                                                        |
+| sidenav-focus      |   [Docs][95] |   -          | [autofocus][96] only supports focusing the first focusable element.|
+| switch             |   [Docs][61] |   [Docs][14] | `mat-slide-toggle`                                     |
+| slider             |   [Docs][62] |   [Docs][16] |                                                        |
+| slider-container   |   [Docs][97] |   -          | Not implemented                                        |
+| subheader          |   [Docs][98] |   [Docs][99] | `matSubheader`                                         |
+| swipe              |   [Docs][100]|   -          | See [HammerJS setup][101] and [Hammer.Swipe][102]      |
+| tabs               |   [Docs][64] |   [Docs][13] |                                                        |
+| textarea           |   [Docs][52] |   [Docs][5]  |                                                        |
+| toolbar            |   [Docs][66] |   [Docs][7]  |                                                        |
+| tooltip            |   [Docs][67] |   [Docs][18] |                                                        |
+| truncate           |   [Docs][103]|   -          | Not implemented                                        |
+| virtual-repeat     |   [Docs][68] |   [Docs][40] | `cdk-virtual-scroll-viewport` and `*cdkVirtualFor`     |
+| whiteframe         |   [Docs][104]|  [Guide][105]| Based on classes and mixins                            |
+
+### Services
+
+| Service          | Old          | New          | Special Notes                                          |
+|------------------|--------------|--------------|--------------------------------------------------------|
+| $mdAriaProvider  |   [Docs][124]|   -          | Not implemented                                        |
```

Comment:
It would be good to say here something like "Use a tool like aXe or Chrome DevTool's Accessibility audit instead"

Reply:
Yeah, I think that we need another section below that explains these in more detail. Then we can change this `Not implemented` for many of these to `See below`.